php/README.md

```diff
@@ -0,0 +1,51 @@
+## Quick Example for a PHP Worker (4 minutes)
+
+This example will show you how to include dependencies with your worker so it will work right when you run it
+remotely on IronWorker.
+
+Note: You'll need PHP installed on your machine to use this example (And [composer](https://getcomposer.org/) for dependencies).
+
+Install the dependencies to your system.
+```sh
+composer install
```

Comment:
yes, in current directory. more precisely, creates `vendor` directory with `autoload.php` and modules. see https://github.com/iron-io/dockerworker/pull/3/files#diff-6ca425180a79e0cd4579cd2c2b785d42R3